stacks-signer/src/ping/mod.rs

```diff
@@ -0,0 +1,75 @@
+use serde_derive::{Deserialize, Serialize};
+mod processor;
+mod runloop;
+use crate::{
+    stacks_client::{PING_SLOT, SLOTS_PER_USER},
+    utils::ToSlotId,
+};
+use uuid::Uuid;
+
+pub use processor::RTTProcessor;
+pub use runloop::{Command, RunLoop};
+
+impl<'a> ToSlotId for &'a Packet {
+    fn slot_id(&self, signer_id: u32) -> u32 {
+        SLOTS_PER_USER * signer_id + PING_SLOT
+    }
+}
+
+/// What is written to the ping slot.
+#[derive(Serialize, Deserialize, Clone)]
+pub enum Packet {
+    /// Outgoing
+    Ping(Ping),
+    /// Incoming
+    Pong(Pong),
+}
+
+#[derive(Serialize, PartialEq, Clone, Deserialize, Debug)]
+/// A ping in a slot means someone has requested you to push a Pong into your slot.
+pub struct Ping {
+    id: u64,
+}
+#[derive(Serialize, Deserialize, Debug, Clone)]
+/// A pong in a slot means someone has responded to an RTT request.
+pub struct Pong {
+    id: u64,
+}
+
+impl From<Pong> for Packet {
+    fn from(value: Pong) -> Self {
+        Self::Pong(value)
+    }
+}
+
+impl From<Ping> for Packet {
+    fn from(value: Ping) -> Self {
+        Self::Ping(value)
+    }
+}
+
+impl Ping {
+    /// Uniquely identify the RTT request
+    pub fn new() -> Self {
+        Ping {
+            id: Uuid::new_v4().as_u64_pair().0,
```

Comment:
This is overkill.  Can you just generate a random number here?  And for the payload, just fill it with random numbers.

Reply:
ack